Campus Density/DiningCells/MealFilterCell.swift

```diff
@@ -69,19 +79,19 @@ class MealFilterCell: UICollectionViewCell {
         }
         filterButtons.removeAll()
 
-        for (index, meal) in mealModel.meals.enumerated() {
+        for meal in mealModel.meals {
             let button = UIButton()
-            button.tag = index
             button.setTitle(mealLabel(meal: meal), for: .normal)
-            button.titleLabel?.font = .sixteen
+            button.titleLabel?.font = .eighteen
             button.setTitleColor(.warmGray, for: .normal)
             button.clipsToBounds = true
-            if meal == mealModel.selectedMeal {
-                button.setTitleColor(.grayishBrown, for: .normal)
-                button.layer.borderColor = UIColor.warmGray.cgColor
-                button.layer.borderWidth = 1
+            if (meal == mealModel.selectedMeal) {
+                button.setTitleColor(.densityGreen, for: .normal)
+                button.titleLabel?.font = .eighteenBold
+                button.layer.borderColor = UIColor.densityGreen.cgColor
+                button.layer.borderWidth = 0
```

Comment:
Lines setting border width to 0, corner radius to 0, border color can be removed. Also don't need parentheses around `meal == mealModel.selectedMeal`